src/ripple/app/ledger/tests/common_ledger.cpp

```diff
@@ -435,28 +435,61 @@ trust(TestAccount& from, TestAccount const& issuer,
 void
 close_and_advance(Ledger::pointer& ledger, std::shared_ptr<Ledger const>& LCL)
```

Comment:
It's true that _common_ledger_ is deprecated, but it's also providing the implementation of `jtx::advance` at the moment.  See https://github.com/ripple/rippled/blob/develop/src/ripple/test/jtx/impl/advance.cpp#L32.  From my perspective, until `close_and_advance` functionality is moved into `jtx`.  keeping all of `close_and_advance` together in one place will make it easier to move in the future.